data/presets/emergency/defibrillator.json

```
@@ -4,12 +4,12 @@
         "indoor",
         "ref",
         "operator",
-        "defibrillator/location"
+        "defibrillator/location",
+        "access_simple"
     ],
     "moreFields": [
         "level",
-        "opening_hours",
-        "access_simple"
+        "opening_hours"
```

Interestingly, https://github.com/openstreetmap/id-tagging-schema/commit/1e0990a961246b7520e0b935325a995a16d6df74 (#359) seems to have been lost on the `main` branch. Should it be re-added in this PR?

```suggestion
        "opening_hours",
        "wikimedia_commons"
```

---

Ah, I see it is included in the `sidewalk` branch: https://github.com/openstreetmap/id-tagging-schema/blame/sidewalk/data/presets/emergency/defibrillator.json

But was removed again in the [`tram_crossings`](https://github.com/openstreetmap/id-tagging-schema/blame/tram_crossings/data/presets/emergency/defibrillator.json) and [`k-yle-wilderness-hut-fields`](https://github.com/openstreetmap/id-tagging-schema/blame/k-yle-wilderness-hut-fields/data/presets/emergency/defibrillator.json) branches via https://github.com/openstreetmap/id-tagging-schema/commit/7b2528664c70a3f754255278ad8a941a3a197803.